recommendations/choices.py

```diff
@@ -0,0 +1,255 @@
+    ("tl", "Tagalog"),
+    ("lb", "Letzeburgesch"),
+    ("ts", "Tsonga"),
+    ("ml", "Malayalam"),
+    ("vo", "Volapük"),
+    ("zu", "Zulu"),
+    ("os", "Ossetian; Ossetic"),
+    ("sm", "Samoan"),
+    ("za", "Zhuang"),
+    ("bn", "Bengali"),
+    ("cu", "Slavic"),
+    ("ga", "Irish"),
+    ("gv", "Manx"),
+    ("hu", "Hungarian"),
+    ("jv", "Javanese"),
+    ("kr", "Kanuri"),
+    ("km", "Khmer"),
+    ("ky", "Kirghiz"),
+    ("na", "Nauru"),
+    ("nr", "Ndebele"),
+    ("oc", "Occitan"),
+    ("ro", "Romanian"),
+    ("ru", "Russian"),
+    ("hy", "Armenian"),
+    ("ch", "Chamorro"),
+    ("xx", "No Language"),
+    ("ba", "Bashkir"),
+    ("gl", "Galician"),
+    ("io", "Ido"),
+    ("lu", "Luba-Katanga"),
+    ("mh", "Marshall"),
+    ("mg", "Malagasy"),
+    ("mo", "Moldavian"),
+    ("mn", "Mongolian"),
+    ("nd", "Ndebele"),
+    ("no", "Norwegian"),
+    ("pl", "Polish"),
+    ("sw", "Swahili"),
+    ("tg", "Tajik"),
+    ("to", "Tonga"),
+    ("wa", "Walloon"),
+    ("yi", "Yiddish"),
+    ("en", "English"),
+    ("as", "Assamese"),
+    ("gd", "Gaelic"),
+    ("kl", "Kalaallisut"),
+    ("my", "Burmese"),
+    ("qu", "Quechua"),
+    ("sn", "Shona"),
+    ("uk", "Ukrainian"),
+    ("fa", "Persian"),
+    ("ka", "Georgian"),
+    ("gu", "Gujarati"),
+    ("av", "Avaric"),
+    ("ae", "Avestan"),
+    ("gn", "Guarani"),
+    ("mt", "Maltese"),
+    ("ne", "Nepali"),
+    ("sv", "Swedish"),
+    ("tt", "Tatar"),
+    ("wo", "Wolof"),
+    ("cn", "Cantonese"),
+    ("cv", "Chuvash"),
+    ("da", "Danish"),
+    ("dz", "Dzongkha"),
+    ("ny", "Chichewa; Nyanja"),
+    ("rn", "Rundi"),
+    ("st", "Sotho"),
+    ("tk", "Turkmen"),
+    ("uz", "Uzbek"),
+    ("vi", "Vietnamese"),
+    ("el", "Greek"),
+    ("ca", "Catalan"),
+    ("cy", "Welsh"),
+    ("de", "German"),
+    ("ks", "Kashmiri"),
+    ("ms", "Malay"),
+    ("nb", "Norwegian Bokmål"),
+    ("rm", "Raeto-Romance"),
+    ("si", "Sinhalese"),
+    ("es", "Spanish"),
+    ("te", "Telugu"),
+    ("tw", "Twi"),
+    ("ps", "Pushto"),
+    ("bg", "Bulgarian"),
+    ("mk", "Macedonian"),
+    ("ik", "Inupiaq"),
+    ("ko", "Korean"),
+    ("lt", "Lithuanian"),
+    ("om", "Oromo"),
+    ("se", "Northern Sami"),
+    ("so", "Somali"),
+    ("ta", "Tamil"),
+    ("ur", "Urdu"),
+    ("am", "Amharic"),
+    ("bs", "Bosnian"),
+    ("dv", "Divehi"),
+    ("ay", "Aymara"),
+    ("bm", "Bambara"),
+    ("ii", "Yi"),
+    ("ie", "Interlingue"),
+    ("kv", "Komi"),
+    ("ku", "Kurdish"),
+    ("nn", "Norwegian Nynorsk"),
+    ("zh", "Mandarin"),
+    ("he", "Hebrew"),
+    ("ee", "Ewe"),
+    ("fj", "Fijian"),
+    ("ff", "Fulah"),
+    ("ht", "Haitian; Haitian Creole"),
+    ("hr", "Croatian"),
+    ("ia", "Interlingua"),
+    ("kj", "Kuanyama"),
+    ("lo", "Lao"),
+    ("lg", "Ganda"),
+    ("mr", "Marathi"),
+    ("sd", "Sindhi"),
+    ("be", "Belarusian"),
+    ("ho", "Hiri Motu"),
+    ("kg", "Kongo"),
+    ("sl", "Slovenian"),
+    ("yo", "Yoruba"),
+]
+
+TRIGGERS = [
```

Comment:
These trigger categories are okay for now (not sure if there's a bigger idea for this) but I think we should definitely utilize the specific topics instead of the categories as a whole. Also we can/should remove some of the topics/categories (For example remove 'A dragon dies' topic from the Animal category and remove the 'Loss/Creepy Crawly' categories).

Reply:
the idea is for the form - pick what triggers you. i agree though, we probably need the EXACT thing cause, i doubt someone would want to just mark 'animal'